pkg/apply/prune/prune.go

```diff
@@ -164,8 +164,12 @@ func (po *PruneOptions) GetPruneObjs(inv inventory.InventoryInfo,
 	pruneObjs := []*unstructured.Unstructured{}
 	for _, pruneID := range pruneIds {
 		pruneObj, err := po.GetObject(pruneID)
+		// If we can not retrieve the object--do not attempt to prune it.
+		// TODO(seans): distinguish between "not found" and other errors.
 		if err != nil {
-			return nil, err
+			klog.V(4).Infof("error (%s) retrieving prune obj (%s/%s)--skip",
```

Comment:
Why are we punting on distinguishing between NotFound an other errors?

Reply:
Resolved by skipping not found object, and return fatal error for other errors as discussed offline.